libnmstate/nm/connection.py

```diff
@@ -255,6 +255,16 @@ def waitfor_active_connection_async(self, ac):
                 'state-changed', self._waitfor_active_connection_callback, ac
             )
         )
+        ac.device_handlers.add(
+            ac.nmdevice.connect(
+                'state-changed', self._wait_device_state_change_callback, ac
+            )
+        )
+
+    def _wait_device_state_change_callback(
```

Comment:
The other handler is named `waitfor...` instead of just `wait...`. Makes sense to keep it uniform.